llvm/lib/Transforms/Yk/StackMaps.cpp

```diff
@@ -22,6 +22,13 @@
 
 #define DEBUG_TYPE "yk-stackmaps"
 
+// The first stackmap ID available for use by this pass.
+//
+// It's 1 because 0 is reserved for the control point.
+//
+// This will have to change when we support >1 control point.
+const int NonCPStackmapIDStart = 1;
+
```

Comment:
Earlier we used:

```
// Stackmap ID zero is reserved for the control point.
//
// This will need to change when we support >1 control point.
```

I suggest using similar phrasing here and/or pointing readers at the other constant (since they're inverses of each other).

Reply:
Fixed in a47d09a8b9695617f6378b4fc72f04611c7fb8df